taiga_halo2/src/shielded_ptx.rs

```diff
@@ -66,12 +69,48 @@ struct ShieldedPartialTransactionProxy {
     inputs: Vec<NoteVPVerifyingInfoSet>,
     outputs: Vec<NoteVPVerifyingInfoSet>,
     binding_sig_r: pallas::Scalar,
+    hints: Vec<u8>,
```

Comment:
What is `hints`?

Reply:
It's Jeremy's requirement.
`hints` describes intents or partial tx to help the solver match. It will be handled outside of Taiga.